pyprecag/kriging_ops.py

```diff
@@ -27,25 +31,152 @@
 # Set default value for vesper_exe
 vesper_exe = r"C:\Program Files (x86)\Vesper\Vesper1.6.exe"
 
+
 def test_for_windows():
     if platform.system() != 'Windows':
         raise IOError("Kriging currently only available on Windows with Vesper installed")
 
+
+# consider enums - https://stackoverflow.com/questions/36932/how-can-i-represent-an-enum-in-python
+# The codes for the variogram. The key is the tag used in the control file_csv
+VESPER_OPTIONS = {"jigraph": {"Don't Show": 0, "Show": 1},
+                  "jimap": {"Don't Show": 0, "Show": 1},
+                  "jlockrg": {"Local": 1, "Global": 0},
+                  "jpntkrg": {"Block": 0, "Point": 1, "Punctual": 1},
```

Comment:
Are there different uses between Point and Punctual?

Reply:
Not that I know of, VESPER dialog has it as Punctual, but the Vesper manual control file description calls it point both use 1.